tests/unit/test_timeframes.py

```diff
@@ -36,20 +36,20 @@
         7.25,
         10.75,
     ],
-    (3, 3, 0, 5, 0): [-0.77777778, 4.13333333, 4.13333333, 5.51666667, 5.05952381],
+    (3, 3, 0, 5, 0): [-1.66666667, 4.13333333, 4.13333333, 5.51666667, 3.01666667],
```

Comment:
Why do these differences at the edges occur?

Reply:
Not sure. I did the calculations by hand and am pretty sure that these numbers are correct for linear extrapolation. Perhaps with constant extrapolation you would get the other numbers? I thought we'd tried to implement linear extrapolation last time so maybe there was simply a bug...

Reply:
Ok, so if these follow the edge point extrapolation as described in the notebook, everything's fine.